requirements.txt

```diff
@@ -0,0 +1,5 @@
+Django>=2.1.3,<2.2.0
```

Comment:
Better use fixed versions. This will guarantee that your app has a determined state.
Version ranges are useful for libraries in order to be flexible when combined with other libraries.

Reply:
Ok, in the course it was explained that this way you will get compatible version (first 2 digits are the same) but with bug fixes & security updates (highest last digit)